src/world/ObjectMgr.cpp

```diff
@@ -3898,4 +3898,172 @@ void ObjectMgr::LoadAreaTrigger()
     while (result->NextRow());
 
     Log.Success("AreaTrigger", "Loaded %u area trigger teleport definitions", count);
+}
+
+void ObjectMgr::LoadEventScripts()
+{
+    Log.Notice("ObjectMgr", "Loading Event Scripts...");
+
+    QueryResult* result = WorldDatabase.Query("SELECT * FROM event_scripts WHERE event_id > 0 ORDER BY event_id");
```

Comment:
Passing the string directly to the function can make the code harder to read, especially if the string is very long. Store the string in a constant variable and pass the variable in to the function instead.

```
const char* eventScriptsQuery = "SELECT * FROM event_scripts WHERE event_id > 0 ORDER BY event_id");
bool success = false;
auto result = WorldDatabase.Query(&success, eventScriptsQuery);
```